src/sentry/static/sentry/app/components/errorRobot.tsx

```diff
@@ -9,6 +9,8 @@ import withApi from 'app/utils/withApi';
 import {Client} from 'app/api';
 import {LightWeightOrganization, Project} from 'app/types';
 import {defined} from 'app/utils';
+import space from 'app/styles/space';
+import robotBackground from 'app/../images/spot/sentry-robot.png';
```

Comment:
Is `errorRobot` lazy loaded?

Reply:
I've added lazy loading to the robot, please double check